src/mixins/itext_behavior.mixin.js

```diff
@@ -473,6 +473,7 @@
         this.hiddenTextarea.selectionStart = newSelection.selectionStart;
         this.hiddenTextarea.selectionEnd = newSelection.selectionEnd;
       }
+      this.hiddenTextarea.setAttribute('dir', this.direction);
```

Comment:
We might want to check if the direction has changed on the object since the textarea was updated somewhere and run an update if so